Analysis/DataModel/include/AnalysisDataModel/HFCandidateSelectionTables.h

```diff
@@ -39,4 +39,14 @@ DECLARE_SOA_COLUMN(IsSelJpsiToEE, isSelJpsiToEE, int);
 } // namespace hf_selcandidate_jpsi
 DECLARE_SOA_TABLE(HFSelJpsiToEECandidate, "AOD", "HFSELJPSICAND", hf_selcandidate_jpsi::IsSelJpsiToEE);
 } // namespace o2::aod
+
+namespace o2::aod
+{
+namespace hf_selcandidate_xic
+{
+DECLARE_SOA_COLUMN(IsSelXicpKpi, isSelXicpKpi, int);
+DECLARE_SOA_COLUMN(IsSelXicpiKp, isSelXicpiKp, int);
+} // namespace hf_selcandidate_xic
+DECLARE_SOA_TABLE(HFSelXicpKpiCandidate, "AOD", "HFSELXICCAND", hf_selcandidate_xic::IsSelXicpKpi, hf_selcandidate_xic::IsSelXicpiKp);
```

Comment:
These should be `IsSelXicToPKPi` etc. to follow the camelCase convention and to match the `DecayType` names.
@DelloStritto could you please add these in your Lc PR as well?

Reply:
Yes, I will. Thanks

Reply:
I am going to do it as well, I will let you know when I conclude.

Reply:
Thank

Reply:
@vkucera I implemented the requested syntax fixes.

Reply:
@mfaggin Thanks. Can you fix the rest as well?

Reply:
@vkucera done also for other Xic methods (see commit 38af94a , clang checks are queued, I will fix possible syntax errors tomorrow, sorry I need to leave now)

Reply:
@vkucera I see that the clang checks are ok now, so the requested modifications are in place.